### ISSUE TITLE
webhook received malformed payload not to channels

### ISSUE THREAD

**Issue by flammie** [apertium/phenny#591]
I think that the message:
```
08:34  begiak> Webhook received malformed payload
```
is not very useful for the people on channel and should probably be sent to admins directly instead or something.

**Comment by scoopgracie** [comment]
Good idea. Sometime I might rewrite my error logger to not use Flask, then
we could add this to it.

**Comment by scoopgracie** [comment]
That message also makes it easy to make a spambot with a simple bash loop
and curl command.

**Comment by unhammer** [comment]
https://github.com/apertium/phenny/blob/6b15a463f050c6acbb205efd6d43a39ffffc9a08/modules/git.py#L135

seems simple to fix! but who's actually running begiak? @jonorthwash ?
